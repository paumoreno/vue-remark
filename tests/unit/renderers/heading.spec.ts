import VueRemarkHeading from "@/renderers/heading.vue";
import { mount } from "@vue/test-utils";

describe(VueRemarkHeading.name, () => {
  test("snapshot", () => {
    const wrapper = mount(VueRemarkHeading, {
      propsData: {
        depth: 2,
      },
      slots: {
        default: "Heading 2",
      },
    });

    expect(wrapper).toMatchSnapshot();
  });
});
