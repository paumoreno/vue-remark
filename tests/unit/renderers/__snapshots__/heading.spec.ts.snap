// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VueRemarkHeading snapshot 1`] = `<h2>Heading 2</h2>`;
